modules/effects/arrow-effects/src/main/kotlin/arrow/effects/IORunLoop.kt

```diff
@@ -233,8 +233,8 @@ internal object IORunLoop {
 
           bFirst = { c: Any? -> IO.just(c) }
 
-          currentIO = IO.async { TODO, cc ->
-            loop(localCont, conn, cc.asyncCallback(currentCC), null, null, null)
+          currentIO = IO.async { _, callback ->
```

Comment:
I think this one should be ignored. As in: no matter what connection the new async gets passed, it'll reuse the one passed in for the current `loop()` call. Can't see a reason to use a modified one or anything here, since there's no new cancellation reasons. Does it make sense to you @pakoito ?

Reply:
Add a comment there, just in case, to explain your reasoning.